ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
+        do {
+            if (reviewStep.reverseListing) {
+                nextStep = [self stepAfterStep:nextStep];
+            } else {
+                nextStep = [self stepBeforeStep:nextStep];
+            }
+            if (nextStep && ![nextStep isKindOfClass:[ORKReviewStep class]]) {
+                [steps addObject:nextStep];
+            }
+        } while (nextStep || [nextStep isKindOfClass:[ORKReviewStep class]]);
+    }
+    if (!reviewStep.reverseListing) {
+        steps = [[NSMutableArray alloc] initWithArray:[[steps reverseObjectEnumerator] allObjects]];
+    }
+    NSMutableArray *filteredSteps = [[NSMutableArray alloc] init];
+    [steps enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
```

Comment:
Much of this filtering logic is no longer needed, as I decided to drop support for filtering single steps during review. I moved the rest of this code to `ORKReviewStepViewController`